threedi_api_client/api.py

```diff
@@ -85,14 +85,32 @@ def __init__(self, env_file=None, config=None, version="v3", asynchronous=False)
         else:
             user_config = EnvironConfig()
 
-        # Get the config variables
         host = user_config.get("THREEDI_API_HOST")
         username = user_config.get("THREEDI_API_USERNAME")
-        password = user_config.get("THREEDI_API_PASSWORD")
-        if not all(x for x in (host, username, password)):
+        if not all(x for x in (host, username)):
             raise ValueError(
                 "ThreediApi requires the THREEDI_API_HOST, THREEDI_API_USERNAME, "
-                "and THREEDI_API_PASSWORD configuration values."
+                "configuration values."
+            )
+
+        # Get the config variables
+        password = user_config.get("THREEDI_API_PASSWORD")
+        try:
+            access_token = user_config.get("THREEDI_API_ACCESS_TOKEN")
+        except KeyError:
```

Comment:
It is unexpected that the `.get` would raise a `KeyError`, this is not like a dict. I made a ticket to refactor this a bit: https://github.com/nens/threedi-api-client/issues/56

Reply:
Okay, I'll perform the proposed refactor in this PR as well.